armi/tests/refSmallReactorBase.yaml

```diff
@@ -77,45 +77,41 @@ blocks:
         control:
             shape: Circle
             material: B4C
-            Tinput: 600.0
-            Thot: 600.0
+            Tinput: 450.0
+            Thot: 450.0
             id: 0.0
             mult: 61.0
             od: 1.286
-        innerDuct:
+        innerDuct: &component_control_duct
             shape: Hexagon
             material: HT9
             Tinput: 450.0
             Thot: 450.0
             ip: 14.268
             mult: 1.0
             op: 14.582
-        duct: &component_control_duct
-            shape: Hexagon
-            material: HT9
-            Tinput: 450.0
-            Thot: 450.0
-            ip: 15.277
-            mult: 1.0
-            op: 16.28228
-        clad:
+            axially expands downwards: True
```

Comment:
Yeah, good catch. They won't expand. I added the `axially expands downwards` part since they _should_ be there. And those blueprints do get run through the expansion methods in process loading - which is done here https://github.com/terrapower/armi/blob/06b839ef7ceea1c56799d9189017e4c4a4a41d99/armi/reactor/converters/tests/test_axialExpansionChanger.py#L542

But you're right, there's no thermal expansion with `Thot` == `Tinput`

Reply:
I am pretty surprised that all these changes to `refSmallReactorBase.yaml` didn't break unit and integration tests downstream.

Did you test that?

Reply:
@keckler I changed Thot != Tinput so that thermal expansion can happen. 

@john-science they definitely did break a bunch of stuff. I updated the PR description to give motivate the need for these changes. Currently working on putting together PRs in all affected downstream projects.